.github/workflows/schedule.yml

```diff
@@ -1,10 +1,10 @@
-name: Stale Check
+name: Scheduled tasks 🕑
 
 on:
   schedule:
     - cron: '30 1 * * *'
```

Comment:
The SDK update task should run later so that it runs after the SDK is built

Reply:
Okay, so that's the reason for the timing, I didn't know about that.

I believe this is really fragile then (and also a future maintenance problem willing to happen). For these kind of things, I usually use this: https://github.com/marketplace/actions/trigger-workflow-and-wait (sso in SDK repo we trigger an update here).

Either way (and I believe you told me so at some point, but I forgot sorry! 😅) why isn't this with Renovate? This can be a good chance to review that.

Reply:
We were using the `unstable` tag in the package file (which is [not supported](https://github.com/renovatebot/renovate/issues/29303)  by renovate) but that was causing issues. It looks like we could switch to renovate with the follow tag option and overriding stability days.

Reply:
The switch to renovate is done now: https://github.com/jellyfin/jellyfin-web/pull/6156